pages/about.vue

```diff
@@ -9,13 +9,9 @@ const description = 'Find out or about me as a person, my personal history, my b
 useSeoMeta({
   title,
   description,
-  ogTitle: title,
```

Comment:
When you use [Nuxt SEO Experiments](https://nuxtseo.com/experiments/getting-started/installation) through the `@nuxtjs/seo` module, it will automatically copy over title / description to the og equivalent